### DISCUSSION TITLE
Add link doesn't work

### DISCUSSION THREAD

**Discussion by Joypacharya** [stm32duino/Arduino_Core_STM32#2203] [Q&A]
I am using the Arduino IDE.
I tried to install STM32 in my board manager.
Due to a link error, I can't add it.
The link would be:
Error downloading https://github.com/stm32duino/BoardManagerFiles/raw/main/package_stmicroelectronics_index.json

**Comment by fpistm** [comment]
Hi @Joypacharya 
The link is correct. 
The file is hosted on GitHub.  Maybe GitHub had an issue or you have a proxy issue.

**Comment by Joypacharya** [comment]
even try to copy paste same link on web Brower.
It is showing.
This site can’t be reached.
I checked no more proxy set in my ide.

**Comment by fpistm** [comment]
![Screenshot_20231124_074419_Chrome](https://github.com/stm32duino/Arduino_Core_STM32/assets/20641798/17a5ab33-2244-4af6-987a-f6c50c369a40)

Unfortunately I have no idea why you get this issue. It is not link to core issue.

**Comment by Joypacharya** [comment]
![image](https://github.com/stm32duino/Arduino_Core_STM32/assets/23740780/4f3dfd51-4bf0-4398-a92d-b995665c7735)

**Comment by Joypacharya** [comment]
check this three images and let me know.
![image](https://github.com/stm32duino/Arduino_Core_STM32/assets/23740780/a9931b1e-808b-451a-be1e-a5771bd52d53)
![image](https://github.com/stm32duino/Arduino_Core_STM32/assets/23740780/e1364faa-1c79-4e84-9127-a3c21c7c6e05)
![image](https://github.com/stm32duino/Arduino_Core_STM32/assets/23740780/c899d4ce-55a1-4e11-ab54-d8985d9f51da)

**Reply by fpistm** [reply]
The fact you are not able to show the file thanks a browser is already an issue. 
The file is hosted here:
https://github.com/stm32duino/BoardManagerFiles
Arduino requires to get it as raw that's all.
As stated, I can't help more as there is no issue with e this file and it is used daily.
There is no issue on GitHub side as shown by https://www.githubstatus.com/
So while you can't get it on a browser, you will never get it under Arduino.
Maybe, you have some restriction linked to your country or firewall, or antivirus or anything else related to the network (proxy, black list,...) but I can't help on this.

**Comment by Joypacharya** [comment]
Thanks for replying

**Reply by fpistm** [reply]
Welcome, sorry to can't help more.
